consent/helpers.py

```diff
@@ -43,7 +43,7 @@ def consent_provided(user_id, course_id, enterprise_customer_uuid):
     enterprise_course_enrollment = get_enterprise_course_enrollment(user_id, course_id, enterprise_customer)
     provided = enterprise_course_enrollment is not None and enterprise_course_enrollment.consent_available
 
-    if enterprise_course_enrollment is None:
+    if enterprise_course_enrollment is None or not enterprise_course_enrollment.consent_available:
```

Comment:
i think this should actually be `or enterprise_course_enrollment.consent_available is None`, since if it is false, that means they declined consent.